guix/1

```diff
@@ -0,0 +1,8 @@
+NAME="Guix System"
+ID=guix
```

Comment:
According to the distrowatch, this distro is in a fixed-release model, but its os-release file does not contain any `VERSION` key😵‍💫

Reply:
there's a chance this is my fault as I believe I got this from a live install, so a fully installed version might have an ID included. I will double check and revert, but in the case they have no ID value is this file name correct to leave as is?

Reply:
Ok, the installed os-release file is just as barebones! I know from the website this is version 1.4.0 I can update the filename to that if it's preferable

Reply:
Thanks for your time on checking a fully-installed distro! 

> I know from the website this is version 1.4.0 I can update the filename to that if it's preferable

Let's merge it as-is, I try to keep the file names consistent, but honestly, not every distro obeys the rule. 😪